src/pages/workspaces/workspace/notebooks/AnalysisLauncher.js

```diff
@@ -0,0 +1,545 @@
+import * as clipboard from 'clipboard-polyfill/text'
```

Comment:
Wondering if we should make a new `src/pages/workspaces/workspace/analysis` directory and start populating files there?

(Also thinking about where to put new files related to the context launcher.)

Reply:
probably worth it.. theres a ticket I made already to re-sort the util files and the modals scattered across a few different places

Reply:
Updated the description here https://broadworkbench.atlassian.net/browse/IA-2807